backend_skeleton/front.py

```diff
@@ -0,0 +1,34 @@
+# This script will act as a front to communicate with central.
+
+
+import time
+import socket
+import commonio
+
+def front_client():
+    client_socket = commonio.set_socket()
+    print("Backend Reached Succesfuly!")
+    try:
+        while True:
+            print("Type 'quit' to quit.")
+            data = client_socket.recv(1024).decode('utf-8')
+            print(f"Central says: {data}")
+            keyInput = input("Request: ")
```

Comment:
Are you planning to convert this into the script that reads from the sensor?
If so, add a `# TODO: ...` comment.

Reply:
sensor.py reads from the sensor as it is and sends it to Central.py
front.py is the stand in for the frontend that receives data from Central when it makes a request to Central.